tck/methods/key.go

```diff
@@ -0,0 +1,33 @@
+package methods
+
+import (
+	"context"
+	"github.com/hashgraph/hedera-sdk-go/v2"
+)
+
+type KeyService struct {
+}
+
+type KeyParams struct {
+	PrivateKey string `json:"privateKey"`
+}
+
+// GeneratePublicKey generates public key based on provided private key
+func (_ KeyService) GeneratePublicKey(_ context.Context, params KeyParams) string {
+	key, err := hedera.PrivateKeyFromString(params.PrivateKey)
+	if err != nil {
+		return ""
+	}
+	return key.PublicKey().String()
+}
+
+// GeneratePrivateKey generate new private key and return it.
+// NOTE: params object is used only to maintain the communication with jRPC client, which is providing empty object when calling this func
```

Comment:
Incorrect comment. The params have nothing to do with jRPC clients. I guess that `context` was assumed here. That would make me think that there is no specific need to pass `context` everywhere. Can we wrap the functions so `context` isn't passed everywhere?